contracts/0.4.24/nos/NodeOperatorsRegistry.sol

```diff
@@ -67,26 +77,31 @@ contract NodeOperatorsRegistry is INodeOperatorsRegistry, IsContract, AragonApp
         uint256 initialUsedSigningKeys;
     }
 
+    /// @dev The cumulative stats of signing keys of added NodeOperators
+    struct SigningKeysStats {
+        uint64 totalSigningKeys;
```

Comment:
Later in the Stacking Router, the keys available for deposit are calculated as `totalSigningKeys` - `usedSigningKeys`, which is not correct, because we need to use `stakingLimit` instead of `totalSigningKeys`

Reply:
can u re-check, pls